src/Run.js

```diff
@@ -1,3 +1,4 @@
+const _ = require('lodash')
```

Comment:
let's add a semicolon and a comment like `// useful array/object utilities` to explain here